src/app/services/ledger.service.ts

```diff
@@ -309,8 +335,9 @@ export class LedgerService {
           this.resetLedger();
         }
         if (!hideNotifications && !resolved) {
-          this.notifications.sendWarning(`Ledger device locked.  Unlock and open the Nano application`);
+          this.notifications.sendWarning(`Unable to connect to the Ledger device.  Make sure your ledger is unlocked.  Restart the Nano App on your ledger if the error persists`);
```

Comment:
also this is minor but i think we should use capitalized word Ledger so it's clear it refers to the device and not the collection of Nano account chains

```suggestion
          this.notifications.sendWarning(`Unable to connect to the Ledger device.  Make sure your Ledger is unlocked.  Restart the Nano App on your Ledger if the error persists`);
```

Reply:
Agreed